sdk/quantum/Azure.Quantum.Jobs/api/Azure.Quantum.Jobs.netstandard2.0.cs

```diff
@@ -150,12 +150,6 @@ public partial class ProviderStatus
         public string Id { get { throw null; } }
         public System.Collections.Generic.IReadOnlyList<Azure.Quantum.Jobs.Models.TargetStatus> Targets { get { throw null; } }
     }
-    public partial class ProviderStatusList
-    {
-        internal ProviderStatusList() { }
-        public string NextLink { get { throw null; } }
-        public System.Collections.Generic.IReadOnlyList<Azure.Quantum.Jobs.Models.ProviderStatus> Values { get { throw null; } }
-    }
```

Comment:
You most likely need to bump the project version as well. 

FYI - Given that this is a preview we can have these kinds of API breaking changes. Once this reaches GA we can no longer have breaking changes!